addons/l10n_ec/data/l10n_ec_chart_data.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
-    <record id="ec_chart_template" model="account.chart.template">
-      <field name="name">Ecuador - Chart of Accounts</field>
-      <field name="currency_id" ref="base.ECS"/>
-      <field name="bank_account_code_prefix">10.01.02</field>
-      <field name="cash_account_code_prefix">10.01.01</field>
-      <field name="transfer_account_code_prefix">10.01.03</field>
-      <field name="code_digits" eval="6"/>
-    </record>
+  <record id="ec_chart_template" model="account.chart.template">
```

Comment:
please don't change the indentation, try to keep the diff minimal

Reply:
@mart-e  Sorry It was my VS editor plugin fault and I dind't notice. I'll correct it.